src/quacc/calculators/mrcc/io.py

```diff
@@ -0,0 +1,200 @@
+from __future__ import annotations
+
+from io import StringIO
+from pathlib import Path
+from typing import TYPE_CHECKING, TypedDict
+
+from ase.io import read
+from ase.units import Hartree
+
+if TYPE_CHECKING:
+    from ase.atoms import Atoms
+
+    class ParamsInfo(TypedDict):
+        mrccinput: dict[str, str]
+        mrccblocks: str
+        charge: int
+        mult: int
+
+    class EnergyInfo(TypedDict):
+        energy: float | None
```

Comment:
Can `energy` ever be `None`?

Reply:
If the calculation finishes, energy should always be given. It would either be the DFT energy or the total energy from a correlated wave-function calculation.